src/Revise.jl

```diff
@@ -319,7 +319,11 @@ function parse_source!(md::ModDict, file::AbstractString, mod::Module, path)
         warn("omitting ", file, " from revision tracking")
         return false
     end
-    parse_source!(md, readstring(file), Symbol(file), 1, mod, path)
+    if VERSION >= v"0.7.0-DEV"
```

Comment:
Thanks!

It's likely you'll have to be more specific about this version number (just using "DEV" has generated complaints in the past). If you can find the commit sha at which this change occurred (check the julia git log), then the `contrib/commit-name.sh` script in julia should give you the version number.